draft-ietf-taps-transport-security.md

```diff
@@ -676,63 +676,75 @@ in the clear. Everything else is encrypted.
 
 - An unreliable transport protocol such as UDP.
 
-# Common Transport Security Features
+# Security Features and Transport Dependencies
 
 There exists a common set of features shared across the transport protocols surveyed in this document.
-The mandatory features should be provided by any transport security protocol, while the optional features
-are extensions that a subset of the protocols provide. For clarity, we also distinguish between handshake
-and record features.
+Mandatory features constitute a baseline of functionality that an application may assume for any TAPS
+implementation. Optional features by contrast may be available to an application depending on application
+and transport dependencies. Applications learn of and use optional features by querying for their presence 
```

Comment:
I'm thinking "Optional features by contrast may vary from implementation to implementation, and so an application cannot simply assume they are available."